src/types/ErrorMessage.ts

```diff
@@ -0,0 +1,10 @@
+export enum ErrorMessage {
+  NONE = '',
+  LOAD = 'Unable to load the todos',
+  TITLE = 'Title can\'t be empty',
+  ADD = 'Unable to add the todo',
+  DELETE = 'Unable to delete the todo',
+  DELETE_ALL = 'Unable to delete the todos',
+  UPDATE = 'Unable to update the todo',
```

Comment:
Usually, we do not store all the text inside the enum, enum is used for a short list of values
Display different text inside component depending on this enum